helpers/mathjax.js

```diff
@@ -1,5 +1,26 @@
 let mathJaxLoaded;
 
+export const htmlBlockMathRenderer = {
+	prepare: async elem => {
```

Comment:
`prepare` performs necessary checks, loads dependencies, and returns `true`/`false` indicating whether replacements (in this case, math typesetting) are required.  

The reason why this is done separately from `render` is - so that all renderers can load their dependencies or perform their async checks in parallel.  In contrast, `render` must be done in series since they operate on the same DOM.